test/jsApi.yml

```diff
@@ -0,0 +1,130 @@
+    -   in: curl -s -X GET "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"GET"'
+#    -   in: curl -s -X HEAD "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+#        out: '"HEAD"'
+    -   in: curl -s -X POST "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"POST"'
+    -   in: curl -s -X PUT "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"PUT"'
+    -   in: curl -s -X DELETE "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"DELETE"'
+#    -   in: curl -s -X CONNECT "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+#        out: '"CONNECT"'
+    -   in: curl -s -X OPTIONS "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"OPTIONS"'
+#    -   in: curl -s -X TRACE "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+#        out: '"TRACE"'
+    -   in: curl -s -X PATCH "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"PATCH"'
+    -   in: bn remove returnMethod
+- test: Test js data input
+  steps:
+    -   in: bn create node8 returnReqBody
+    -   in: |-
+            echo 'exports.handler = (_, req) => req.body;' >function.js
+    -   in: bn deploy returnReqBody
+    -   in: curl -s "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnReqBody"
+        out: '{"type":"Buffer","data":[]}'
+    -   in: curl -s --data '{' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnReqBody"
+        out: '{"type":"Buffer","data":[123]}'
+    -   in: curl -s --data '{}' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnReqBody"
+        out: '{"type":"Buffer","data":[123,125]}'
+    -   in: curl -s --data ' ' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnReqBody"
+        out: '{"type":"Buffer","data":[32]}'
+    -   in: curl -s --data-urlencode ' ' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnReqBody"
+        out: '{"type":"Buffer","data":[37,50,48]}'
+    -   in: echo | curl -s --data-binary @- "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnReqBody"
+        out: '{"type":"Buffer","data":[10]}'
+    -   in: echo | gzip -n | curl -s --data-binary @- "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnReqBody"
+        out: '{"type":"Buffer","data":[31,139,8,0,0,0,0,0,0,3,227,2,0,147,6,215,50,1,0,0,0]}'
+    -   in: bn remove returnReqBody
+- test: Test js headers input
+  steps:
+    -   in: bn create node8 returnReqTestHeader
+    -   in: |-
+            echo 'exports.handler = (_, req) => req.headers.test;' >function.js
+    -   in: bn deploy returnReqTestHeader
+    -   in: curl -s "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnReqTestHeader"
+        out: ''
+    -   in: |-
```

Comment:
Mostly just a preference question. Why use `|-` operator instead of single quoting the curl statements?